public/app/plugins/datasource/mssql/specs/datasource.test.ts

```diff
@@ -156,6 +156,50 @@ describe('MSSQLDatasource', () => {
     });
   });
 
+  describe('When performing metricFindQuery without key, value columns', () => {
+    let results: any;
+    const query = 'select id, values from atable';
+    const response = {
+      results: {
+        tempvar: {
+          refId: 'tempvar',
+          frames: [
+            dataFrameToJSON(
+              new MutableDataFrame({
+                fields: [
+                  { name: 'id', values: [1, 2, 3] },
+                  { name: 'values', values: ['test1', 'test2', 'test3'] },
+                ],
+                meta: {
+                  executedQueryString: 'select id, values from atable',
+                },
+              })
+            ),
+          ],
+        },
+      },
+    };
+
+    beforeEach(() => {
+      fetchMock.mockImplementation(() => of(createFetchResponse(response)));
+
+      return ctx.ds.metricFindQuery(query).then((data: any) => {
+        results = data;
+      });
+    });
+
+    it('should return list of all field values as text', () => {
+      expect(results).toEqual([
+        { text: 1 },
```

Comment:
A bit unsure if it makes sense to mix data types. But looked like old code did this